appnexus/lucid
# 703 optimizations

## PR Checklist

- Manually tested across supported browsers
  - [ ] IE11 (Win 7)
  - [ ] Edge (Win 10)

fixes #703 (sorta, i.e. it does make mergeProps maintain references, but not omitProps. omitProps is trickier, because the component calling it would have to make sure to use static function references instead of anonymous functions. But I think the general memoization of selectors sidesteps the issue in a lot of ways since it will avoid even getting to a component's render if its props haven't changed)
